orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -670,7 +670,7 @@ def test_op_scalar(self, math_sign_ops):
     @parameterized.expand(binary_ops, name_func=rename_func)
     def test_op_binary_tensor(self, binary_op, op_sign, alpha_supported):
         device = self.get_device()
-        cpu_input = torch.rand(3, 3)
+        cpu_input = torch.rand(3, 1)  # use broadcasting in the second dim.
```

Comment:
Do we want a broadcast and non-broadcast test case?

Reply:
I don't think non-broadcast adds additional coverage, so I did not add it. If you feel it should be there, I can add it.

Reply:
I think we could add a little more test coverage of exercising the broadcast calculation. I know we copied broadcast dimension calculation from elsewhere in the code (so hopefully it works), but we don't actually have much tests of this specific copy of the broadcast code. I don't think it has to be in _this_ function, but I don't think it would hurt to exercise a couple of broadcast calculation shapes?

Just something to consider.

Reply:
@jamill - I added a test for broadcasting. Happy to add more cases or let me know if you think that is a good start for coverage. Thank you!